File: C/easy/e039.c
/* 31/12/2016 */
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char* argv[]){
    int n = (int) strtol(argv[1], NULL, 10);

    for(int i = 1; i <= n; i++){
        if(i%3 == 0 && i%5 == 0){
            printf("FizzBuzz\n");
        }else if(i%3 == 0){
            printf("Fizz\n");
        }else if(i%5 == 0){
            printf("Buzz\n");
        }else{
            printf("%d\n", i);
        }
    }
}

File: JavaScript/easy/e039.js
/* 12/04/2016 */

function fizzBuzz(n){
  for(var i=1; i <= n; i++)
    console.log( !(i%3) && !(i%5)  ? "FizzBuzz" : !(i%3) ? "Fizz" : !(i%5) ? "Buzz" : i);
}
